app/controllers/subscription_controller.rb

```diff
@@ -0,0 +1,12 @@
+require 'rake'
+Rake::Task.clear
+TestApp::Application.load_tasks
+
+class SubscriptionController < ApplicationController
+
+  def alert_subscriber
+    Rake::Task[:alert_subscribers].reenable
```

Comment:
what is reenable here?

Reply:
reenable resets the task's already_invoked state, allowing the task to then be executed again.